### ISSUE TITLE
(m)xadsr produces clicks

### ISSUE THREAD

**Issue by tjingboem** [csound/csound#1466]
when making an example for mxadsr and xadsr, i found that xadsr had unexpected clicks.
So i compared adsr to xadsr and for sure, it was xadsr that had abrupt changes from one enevelope state to another
![Screenshot from 2021-04-28 15-26-55](https://user-images.githubusercontent.com/6670911/116412023-8e0f4d80-a836-11eb-87bf-ebf1abfc0f36.png)
...adsr versus xadsr

Perhaps there is something wrong with xadsr and mxadsr, or i am at fault?

```
<CsoundSynthesizer>
<CsOptions>
; Select audio/midi flags here according to platform
-odac    ;;;realtime audio out
;-iadc    ;;;uncomment -iadc if realtime audio input is needed too
; For Non-realtime ouput leave only the line below:
; -o mxadsr.wav -W ;;; for file output any platform
</CsOptions>
<CsInstruments>

; by Menno Knevel - 2021

sr = 44100
ksmps = 32
nchnls = 2
0dbfs  = 1

instr 1
 	 	 	 
kenv	adsr	p4, 0, 1, 0.3    ;change attack time with p4
asig	pluck	kenv, 220, 220, 2, 1	 
	outs	asig, asig
	 
endin
instr 2
 	 	 	 
kenv	xadsr	p4, 1 , 1, 0.3    ;change attack time with p4
asig	pluck	kenv, 220, 220, 2, 1	 
	outs	asig, asig
	 
endin
</CsInstruments>
<CsScore>
f 2 0 4096 10 1	
s
i1 0 2 .01
i1 3 . .5
i1 6 . .9
i1 9 . .00001
s
i2 0 2 .01
i2 3 . .5
i2 6 . .9
i2 9 . .00001
e
</CsScore>
</CsoundSynthesizer>
```

**Comment by tjingboem** [comment]
<img width="972" alt="Screenshot 2021-04-29 at 11 40 16" src="https://user-images.githubusercontent.com/6670911/116534151-77becb80-a8e2-11eb-96b1-a083e6c01916.png">

This is a clear picture from what is happening with the attack time in the adsr (top) and xadsr (bottom).
Thanks Stefano for that.

The attack from xadsr does not seem to reach the level of the decay.
The release seems okay.

The abrupt change from attack to decay produces the click.

**Comment by joel-binarybrain** [comment]
The attack stage looks to be scaled wrong by a factor of ten. It lands on a gain of 0.1 instead of 1.0

Lines 791 and 797 of OOps/ugens1.c

I haven't tried fixing it.

**Comment by joel-binarybrain** [comment]
I changed the scaling and tested again with this csd and it looks like it fixed it. The first and last one still have an issue but this is because the first one is faster than the k-rate allows (which can be solved by using an a-rate envelope or setting ksmps=1) and the last one, the attack time is shorter than 1 sample so it will still produce a click. 

I've opened a pull request on the change.
